icubam/messaging/client.py

```diff
@@ -0,0 +1,29 @@
+from absl import logging
+import os.path
+from tornado import httpclient
+from typing import List, Optional
+
+from icubam.messaging.handlers import onoff
+
+
+class MessageServerClient:
+  """Client for HTTP-based comnunication with the MessageServer"""
+
+  def __init__(self, config):
+    self.config = config
+    self.http_client = httpclient.AsyncHTTPClient()
+
+  async def notify(self,
+                   user_id: int,
+                   icu_ids: List[int],
+                   on: bool = True,
+                   delay: Optional[int] = None):
+    """Notify the scheduler that a user must be added / remove from the loop."""
+    if not icu_ids:
+      return logging.info('nothing to change. Aborting')
+
+    request = onoff.OnOffRequest(user_id, list(icu_ids), on, delay)
+    url = os.path.join(self.config.messaging.base_url,
```

Comment:
I guess this can be set in the constructor as self.url = ....

Reply:
we might have other routes... then the url would be ambiguous i believe